tools/crate-hasher/Cargo.toml

```diff
@@ -16,7 +16,8 @@ opt-level = 0
 [dependencies]
 anyhow = "1.0"
 clap = { version = "~3.1.18", features = ["derive"] }
-ignore = "0.4"
+# pin to 0.4.18 to deal with newer version relying on 1.65.0
+ignore = "0.4.18"
```

Comment:
This doesn't pin it, you need `ignore = "=0.4.18"`.

Reply:
Done in https://github.com/awslabs/smithy-rs/pull/2173/commits/5840533dfaee8d37011457a4e80aef05bbff02af.